help_to_heat/portal/download_views.py

```diff
@@ -138,6 +197,26 @@ def download_feedback_view(request):
     return response
 
 
+@require_http_methods(["GET"])
+@decorators.requires_service_manager
+def download_referrals_all_view(request):
+    feedbacks = portal_models.Referral.objects.all()
+    downloaded_at = timezone.now()
+    file_name = downloaded_at.strftime("%d-%m-%Y %H_%M")
+    response = create_referral_all_xlsx(feedbacks, file_name)
+    return response
+
+
+@require_http_methods(["GET"])
+@decorators.requires_service_manager
+def download_referrals_last_week_view(request):
+    feedbacks = portal_models.Referral.objects.filter(created_at__gte=datetime.now() - timedelta(days=7))
```

Comment:
I only discovered this syntax today and it's SO COOL!?

Reply:
The filter syntax? It's pretty good, yeah :D

Reply:
Oh specifically the date filtering here!

Reply:
Yeah, time deltas are fun.